octobot_trading/exchanges/traders/trader.pxd

```diff
@@ -40,4 +40,5 @@ cdef class Trader(util.Initializable):
     cpdef object set_risk(self, object risk)
     cpdef object convert_order_to_trade(self, object order)
 
-    cdef bint _has_open_position(self, str symbol)
```

Comment:
I don't remember if cpdef works with any()

Reply:
It doenst work, I tried both cdef and cpdef

Reply:
generators don't work with cython. We gotter consumer the generator into a list and use this list into any() for it to be cythonizable. here it would mean doing this:
```
return any([
                position.size
                for position in self.exchange_manager.exchange_personal_data.positions_manager.get_symbol_positions(
                    symbol=symbol
                )])
```
Not sure it's worth it though. Usually doing this is faster and cythonizable:
```
for position in self.exchange_manager.exchange_personal_data.positions_manager.get_symbol_positions(
            symbol=symbol
    ):
    if position.size:
        return True
return False
```

Reply:
I changed it to your second suggestion

Reply:
Do you want to appl those changes or should we merge like this ?